source/documentation/fuseki2/fuseki-config-endpoint.md

```diff
@@ -121,6 +93,61 @@ without a configuration file is:
 | `fuseki:upload` | HTML form file upload |
 | `fuseki:no-op`  | An operation that causes a 400 or 404 error |
 
+Custom extensions can be added (see [Programmatic configuration](fuseki-main.html#build) of the Fuseki server). To be able to uniquely identify the operation, these are usually 
+
+        fuseki:endpoint     [ 
+            fuseki:operation fuseki:shacl ;
+            fuseki:name      "shacl" ;
+        ] ;
+
+See the section ["Integration with Apache Jena Fuseki"](https://jena.apache.org/documentation/shacl/#integration-with-apache-jena-fuseki) for details of the SHACL support. While this operation is part of the standard Fuseki distribution, this operation is added during system initialization, using the custom operation support.
```

Comment:
Should it be a relative link instead of pointing directly to `jena.apache.org`? (for those running the site locally for example?)

Reply:
Good point. We can make it "/documentation/shacl/#integration-with-apache-jena-fuseki" . I like the rooted path because if the Fuseki docs get structured in some way, the "external" link will still work.